prow/spyglass/lenses/links/links.go

```diff
@@ -90,22 +93,56 @@ func humanReadableName(name string) string {
 	return strings.Join(words, " ")
 }
 
+// Config is an optional configuration for links lens.
+type Config struct {
+	GCSBrowserPrefix string `json:"gcs_browser_prefix,omitempty"`
+}
+
+func parseConfig(raw json.RawMessage) (*Config, error) {
+	c := &Config{
+		GCSBrowserPrefix: defaultGCSBrowserPrefix,
+	}
+	if len(raw) == 0 {
+		return c, nil
+	}
+	return c, json.Unmarshal(raw, c)
+}
+
+func clickableLink(url string, c *Config) string {
```

Comment:
Log files are generated by arbitrary untrusted code. Thus the URI can be anything, potentially something malicious. 

Can we do a better job protecting the user from harm here?

1) Make sure the link is sanitized and less likely to be malicious
2) Redirect to something first, so that we do not leak the person's internal URLs to the destination site.
  - AKA a link to google.com in a prow.k8s.io/view/gcs/whatever log will first route through prow.k8s.io/redirect?dest=google.com
  - Otherwise we will be sending view/gcs/whatever to google.com
3) Otherwise (or maybe in addition) putting this functionality behind a config variable that defaults to off

Reply:
I added redirect service. The lens is already optional and requires enabling it in prow's config.